src/test/rpc/Subscribe_test.cpp

```diff
@@ -163,7 +164,7 @@ class Subscribe_test : public beast::unit_test::suite
     }
 
     void
-    testTransactions()
+    testTransactions_APIv1()
```

Comment:
This PR is great and I'm approving it. But I think we should also add `testcase("transactions API Version 1")`. Small change :) I know that a bunch of tests don't have the testcase, and I'm not sure why... but if we have the facility, we should use it 😆